packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -2938,3 +2926,128 @@ class _RenderCupertinoDialogActions extends RenderBox
     return defaultHitTestChildren(result, position: position);
   }
 }
+
+// A column layout with two widgets, where the top widget expands vertically as
+// needed, and the bottom widget has a minimum height.
+//
+// Both child widgets stretch horizontally to the parent's maximum width
+// constraint, with vertical space allocated in this priority:
+//
+//  1. The `bottom` widget receives its requested height, up to a
+//     `bottomMinHeight` limit.
+//  2. The `top` widget receives its requested height.
```

Comment:
This statement is not true right? only if the constant height is large enough to contain both top and effectivebottomminheight

Reply:
I've added the container's constraint to all three bullet points.